### ISSUE TITLE
[Feature Request] volume flyouts

### ISSUE THREAD

**Issue by fxzfdazy** [steam3d/MagicPods-Windows#416]
I like this software; it has a beautiful UI. Can I use it to replace the default Windows volume flyouts?
![屏幕截图 2024-12-10 181036](https://github.com/user-attachments/assets/1f150b53-463c-4b27-bad2-86694f2ff123)

**Comment by steam3d** [comment]
I don't think so, because I haven't made any changes in Windows. MagicPods works in a "sandbox."
